packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -29,18 +29,28 @@ export async function handleQuoting(
     invoiceUrl: payment.intent.incomingPaymentUrl
   })
 
-  if (
-    payment.destinationAccount.scale !== destination.destinationAsset.scale ||
-    payment.destinationAccount.code !== destination.destinationAsset.code
-  ) {
-    deps.logger.warn(
-      {
-        oldAsset: payment.destinationAccount,
-        newAsset: destination.destinationAsset
-      },
-      'asset changed'
-    )
-    throw Pay.PaymentError.DestinationAssetConflict
+  if (payment.destinationAccount) {
+    if (
+      payment.destinationAccount.scale !== destination.destinationAsset.scale ||
+      payment.destinationAccount.code !== destination.destinationAsset.code
+    ) {
+      deps.logger.warn(
+        {
+          oldAsset: payment.destinationAccount,
+          newAsset: destination.destinationAsset
+        },
+        'asset changed'
+      )
+      throw Pay.PaymentError.DestinationAssetConflict
+    }
+  } else {
+    await payment.$query(deps.knex).patch({
+      destinationAccount: {
+        scale: destination.destinationAsset.scale,
+        code: destination.destinationAsset.code,
+        url: destination.accountUrl
+      }
+    })
   }
```

Comment:
Can you elaborate?
An outgoing payment will only become `EXPIRED` if it was never authorized.
(It might be possible to authorize it in the same `PUT` request to re-quote? I'm not sure if create and/or authorize permission is needed to requote.)

Reply:
Ah, nevermind — I misremembered how unfavorable exchange rate changes are handled. I was thinking that they would be requoted, but it is irrevocable instead.